### ISSUE TITLE
Please add __version__ attribute.

### ISSUE THREAD

**Issue by apiszcz** [IntelVCL/Open3D#642]
Please consider adding the __version__ attribute to the base module.

PEP0396
https://www.python.org/dev/peps/pep-0396/

I do not see the attribute:

>>> dir(open3d)
['Always', 'ColorMapOptmizationOption', 'CorrespondenceChecker', 'CorrespondenceCheckerBasedOnDistance', 'CorrespondenceCheckerBasedOnEdgeLength', 'CorrespondenceCheckerBasedOnNormal', 'Debug', 'DoubleVector', 'Error', 'FastGlobalRegistrationOption', 'Feature', 'Gaussian3', 'Gaussian5', 'Gaussian7', 'Geometry', 'Geometry2D', 'Geometry3D', 'GlobalOptimizationConvergenceCriteria', 'GlobalOptimizationGaussNewton', 'GlobalOptimizationLevenbergMarquardt', 'GlobalOptimizationMethod', 'GlobalOptimizationOption', 'Gray32', 'ICPConvergenceCriteria', 'Image', 'ImageFilterType', 'Info', 'IntVector', 'KDTreeFlann', 'KDTreeSearchParam', 'KDTreeSearchParamHybrid', 'KDTreeSearchParamKNN', 'KDTreeSearchParamRadius', 'Kinect2ColorCameraDefault', 'Kinect2DepthCameraDefault', 'LineSet', 'Matrix4dVector', 'None', 'OdometryOption', 'PinholeCameraIntrinsic', 'PinholeCameraIntrinsicParameters', 'PinholeCameraTrajectory', 'PointCloud', 'PoseGraph', 'PoseGraphEdge', 'PoseGraphEdgeVector', 'PoseGraphNode', 'PoseGraphNodeVector', 'PrimeSenseDefault', 'RANSACConvergenceCriteria', 'RGB8', 'RGBDImage', 'RGBDOdometryJacobian', 'RGBDOdometryJacobianFromColorTerm', 'RGBDOdometryJacobianFromHybridTerm', 'RegistrationResult', 'RenderOption', 'ScalableTSDFVolume', 'SelectionPolygonVolume', 'Sobel3dx', 'Sobel3dy', 'TSDFVolume', 'TSDFVolumeColorType', 'TransformationEstimation', 'TransformationEstimationPointToPlane', 'TransformationEstimationPointToPoint', 'TriangleMesh', 'UniformTSDFVolume', 'Vector2iVector', 'Vector3dVector', 'Vector3iVector', 'VerbosityLevel', 'ViewControl', 'Visualizer', 'VisualizerWithEditing', 'VisualizerWithKeyCallback', 'Warning', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'color_map_optimization', 'compute_fpfh_feature', 'compute_point_cloud_mahalanobis_distance', 'compute_point_cloud_mean_and_covariance', 'compute_point_cloud_nearest_neighbor_distance', 'compute_point_cloud_to_point_cloud_distance', 'compute_rgbd_odometry', 'create_image_pyramid', 'create_mesh_arrow', 'create_mesh_cone', 'create_mesh_coordinate_frame', 'create_mesh_cylinder', 'create_mesh_sphere', 'create_point_cloud_from_depth_image', 'create_point_cloud_from_rgbd_image', 'create_rgbd_image_from_color_and_depth', 'create_rgbd_image_from_nyu_format', 'create_rgbd_image_from_sun_format', 'create_rgbd_image_from_tum_format', 'crop_point_cloud', 'crop_triangle_mesh', 'draw_geometries', 'draw_geometries_with_animation_callback', 'draw_geometries_with_custom_animation', 'draw_geometries_with_editing', 'draw_geometries_with_key_callbacks', 'estimate_normals', 'evaluate_registration', 'filter_image', 'filter_image_pyramid', 'get_information_matrix_from_point_clouds', 'get_verbosity_level', 'global_optimization', 'importlib', 'linux', 'open3d', 'orient_normals_to_align_with_direction', 'orient_normals_towards_camera_location', 'read_feature', 'read_image', 'read_pinhole_camera_intrinsic', 'read_pinhole_camera_trajectory', 'read_point_cloud', 'read_pose_graph', 'read_selection_polygon_volume', 'read_triangle_mesh', 'registration_colored_icp', 'registration_fast_based_on_feature_matching', 'registration_icp', 'registration_ransac_based_on_correspondence', 'registration_ransac_based_on_feature_matching', 'select_down_sample', 'set_verbosity_level', 'sys', 'uniform_down_sample', 'voxel_down_sample', 'write_feature', 'write_image', 'write_pinhole_camera_intrinsic', 'write_pinhole_camera_trajectory', 'write_point_cloud', 'write_pose_graph', 'write_triangle_mesh']

**Comment by yxlao** [comment]
We do have that in https://github.com/IntelVCL/Open3D/blob/master/src/Python/Package/open3d/__init__.py#L31

```
➜ python -c "import open3d; print(open3d.__version__)"
0.4.0.0
```

**Comment by apiszcz** [comment]
I used conda for the install, however 0.3.0 is installed, perhaps 0.4 has
the fix? Is there a conda channel to load the latest version/build versus
3.0?

conda install -c open3d-admin open3d
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    open3d-0.3.0               |           py35_0        15.5 MB
open3d-admin

**Comment by yxlao** [comment]
Right, 0.4.0 has the fix
We've uploaded the ubuntu and mac conda package for 0.4.0. Windows conda package will be coming shortly.

**Comment by apiszcz** [comment]
$ conda install -c open3d-admin open3d
Solving environment: done
## Package Plan ##
  environment location: /plib/a3
  added / updated specs:
    - open3d

The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    open3d-0.3.0               |           py35_0        15.5 MB
open3d-admin
The following NEW packages will be INSTALLED:
    open3d: 0.3.0-py35_0 open3d-admin

**Comment by yxlao** [comment]
@apiszcz Seems that you're getting open3d 0.3.0 in py35. I believe there are some python package dependency conflicts in our conda py3.5 builds. The result is that conda install open3d in python 3.5 always retrieves the 0.3.0 version. However 2.7 and 3.6 works for open3d 0.4.0. 

For a quick fix, I'd suggest switching to py 3.6 for now. We'll be working on a proper fix.

**Comment by yxlao** [comment]
Conda py35 dependency conflicts resolved and pushed to Anaconda cloud
